api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirPatientServiceImpl.java

```diff
@@ -115,6 +115,21 @@ public IBundleProvider getPatientEverything(TokenParam patientId) {
 		        .addParameter(FhirConstants.COMMON_SEARCH_HANDLER, FhirConstants.ID_PROPERTY,
 		            new TokenAndListParam().addAnd(patientId));
 		
+		getEverythingReverseInclude(theParams);
+		return searchQuery.getQueryResults(theParams, dao, translator, searchQueryInclude);
+	}
+	
+	@Override
+	@Transactional(readOnly = true)
+	public IBundleProvider getPatientEverythingType() {
+		SearchParameterMap theParams = new SearchParameterMap().addParameter(FhirConstants.EVERYTHING_SEARCH_HANDLER, "");
+		
+		getEverythingReverseInclude(theParams);
+		return searchQuery.getQueryResults(theParams, dao, translator, searchQueryInclude);
+		
+	}
+	
+	private void getEverythingReverseInclude(SearchParameterMap theParams) {
```

Comment:
`getEverythingReverseInclude` feels like the method returns us something.
A better name should be `populateEverythingOperationParams`